.github/workflows/ci.yml

```diff
@@ -3,49 +3,112 @@ name: CI
 on:
   push:
     branches:
+      - develop
```

Comment:
when you do this, your PR will double number of builds from this file, as you both `push` to develop and opened a PR with base = develop.

Reply:
We can adjust the triggers, I was just using the same as the currently existing triggers on in ci.yml.